contrib/demo/testDemos.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+CURRENT_DIR="$(pwd)"
+DEMO_ROOT="$(cd $(dirname "${BASH_SOURCE}") && pwd)"
+KCP_ROOT="$(cd ${DEMO_ROOT}/../.. && pwd)"
+
+source ${DEMO_ROOT}/.startUtils
+setupTraps $0
+
+cleanupDemoOutput() {
+  sed -E --posix -e '/^☸️ \$ /d' -e '/^ +certificate-authority-data: /d' -e '/^ +server: https:\/\/127\.0\.0\.1:[0-9]{5}$/d' -e 's/^(.*)( +[0-9]+s|[0-9]+m([0-9]+s)?)$/\1/' $1
+}
+
+error=false
+for demo in kubecon ; do
```

Comment:
This for loop seems like a premature optimization.

Reply:
My plan is to add the already-existing second demo about API negotiation :-)

Reply:
Done